sass/_theme.scss

```diff
@@ -1,149 +1,103 @@
     }
+}
 
-    .footnote-definition sup {
-        color: $secondary;
-    }
 
-    table {
-        th, td {
-            border-color: darken($tertiary, 5%);
-        }
+// MARK: Light/dark config
 
-        thead, tr:nth-child(even) {
-            background-color: lighten($tertiary, 10%);
-        }
-    }
+.dark-mode-buttons {
+    position: absolute;
+    top: 1em;
+    right: 1em;
 }
 
-@mixin dark-theme {
-    color: $foreground-dark;
-    background-color: $background-dark;
+.dark-mode-button {
+    border: none;
+    background-color: transparent;
 
-    .secondary {
-        color: $secondary-dark;
+    &:hover {
+        cursor: pointer;
     }
+}
 
-    a, a:link, a:visited {
-        color: $accent-dark;
-    }
 
-    a:hover {
-        color: lighten($accent-dark, 10%);
-    }
+// MARK: Styles
 
-    blockquote {
-        border-left: 2px solid $secondary-dark;
-    }
+body {
+    color: var(--foreground);
+    background-color: var(--background);
 
-    code {
-        background-color: $tertiary-dark;
-    }
-
-    pre code {
-        background-color: transparent;
-    }
-
-    .footnote-definition sup {
-        color: $secondary-dark;
+    &:not(.dark-mode) {         
+        #dark-mode-on {
+            display: inline;
+        }
+        #dark-mode-off {
+            display: none;
+        }
     }
 
-    table {
-        th, td {
-            border-color: $tertiary-dark;
+    &.dark-mode {
+        #dark-mode-on {
+            display: none;
         }
-
-        thead, tr:nth-child(even) {
-            background-color: darken($tertiary-dark, 15%);
+        #dark-mode-off {
+            display: inline;
         }
     }
 }
 
-@media (prefers-color-scheme: light) {
-    body {
-        @include light-theme;
-    }
+.secondary {
+    color: var(--secondary);
 }
 
-@media (prefers-color-scheme: dark) {
-    body {
-        @include dark-theme;
-    }
+a, a:link, a:visited {
+    color: var(--accent);
 }
 
-.dark-mode-buttons {
-    position: absolute;
-
-    top: 1em;
-    right: 1em;
+a:hover {
+    color: color-mix(in hsl, var(--accent) 30%, black);
 }
 
-.dark-mode-button {
-    border: none;
-    background-color: transparent;
-
-    &:hover {
-        cursor: pointer;
-    }
+blockquote {
+    border-left: 2px solid var(--secondary);
 }
 
-body:not(.dark-mode) {
-    @include light-theme;
-
-    #dark-mode-on {
-        display: inline;
-    }
+code {
+    background-color: var(--tertiary);
+}
 
-    #dark-mode-off {
-        display: none;
-    }
+pre code {
+    background-color: transparent;
 }
 
-body.dark-mode {
-    @include dark-theme;
+.footnote-definition sup {
+    color: var(--secondary);
+}
 
-    #dark-mode-on {
-        display: none;
+table {
+    th, td {
+        border-color: color-mix(in hsl, var(--tertiary) 5%, black);
     }
 
-    #dark-mode-off {
-        display: inline;
+    thead, tr:nth-child(even) {
+        background-color: color-mix(in hsl, var(--tertiary) 10%, white);
```

Comment:
Rough approximation (old vs. new)
```scss
background-color: lighten($tertiary, 10%);
background-color: color-mix(in hsl, var(--tertiary) 10%, white);
```

[Relative color syntax](https://www.matuzo.at/blog/2023/100daysof-day92/) isn't supported on FFox yet :( so this feels like an okay workaround.